rebar.config

```diff
@@ -29,21 +31,14 @@
 
 {minimum_otp_vsn, "21.0"}.
 
-{escript_emu_args, "%%! -connect_all false \n" }.
+{escript_emu_args, "%%! -connect_all false\n" }.
+%% For some reason we need to explicitly include redbug,
+%% even if it already is a dependency.
+{escript_incl_apps, [redbug]}.
 
-{profiles, [ { debug
-             , [ { deps
-                 , [ {redbug, {git, "https://github.com/massemanet/redbug.git", {ref, "8dd853154799da973b20be3a33fc8da9618e6393"}}}
-                   ]
-                 }
-               , { escript_emu_args
-                 , "%%! -connect_all false -erlang_ls logging_enabled true \n"
-                 }
-                 %% For some reason we need to explicitly include redbug,
-                 %% even if it already is a dependency.
-               , {escript_incl_apps, [redbug]}
-               ]
-             }
+%% Keeping the debug profile as an alias, since many clients were
```

Comment:
In what ways were clients relying on the `debug` profile? Because of the path to the escript?

Reply:
Yes, exactly.